tests/test_ci/TestControllers.py

```diff
@@ -459,3 +504,27 @@ def generate_git_api_header(event, sig):
             ('Content-Type', 'application/json'),
             ('Content-Length', 6615)
         ])
+
+    def add_CCExversion(self, c, version='v2.1', commit='abcdefgh', released='2018-05-30T20:18:44Z'):
```

Comment:
This seems contradictory... You say "add a verson to the DB", but you emulate a full POST request to the webhook... Why not just directly push something into the DB?

Reply:
My mistake; it just fell out of my mind. I've reverted to using direct insertion.